installation/downloadReferences.sh

```diff
@@ -50,7 +50,7 @@ mkdir_cd hg19_GRCh37_1000genomes
 	DBSNP_VERSION=135
 	mkdir_cd databases/dbSNP/dbSNP_$DBSNP_VERSION
 
-	EXPECTED_MD5SUM=fed2a31b5a5d8fe12e072576c0c17199
```

Comment:
The new MD5 does not represent the old file. The file on the server has in fact changed. The dbSNP is only hosted in its most recent version, so the link will always download the newest version, which right now (according to the file header) is version 150. More in my comment below.

Reply:
The line `zcat 00-All.vcf.gz | awk '/^#/{print} /VC=SNV/{ v=$8; sub(/.*dbSNPBuildID=/, "", v); sub(/;.*/, "", v); if (v~/^[0-9]+$/ && int(v)<='$DBSNP_VERSION') print }' | bgzip > 00-All.SNV.vcf.gz` is important. Here, the script filters the entries such that only entries with version number <=135 are accepted. Thus, the entries in the final file are technically of version 135. However, since the header of the original file has changed the MD5 is still broken.